calculators/referendum_expenditures_by_origin.rb

```diff
@@ -0,0 +1,101 @@
+class ReferendumExpendituresByOrigin
+  def initialize(candidates: [], ballot_measures: [], committees: [])
+    @ballot_measures = ballot_measures
+    @committees_by_filer_id =
+      committees.where('"Filer_ID" IS NOT NULL').index_by { |c| c.Filer_ID }
+  end
+
+  def fetch
+    totals = ActiveRecord::Base.connection.execute(<<-SQL)
+      SELECT "Measure_Number", "Sup_Opp_Cd", sum("Amount") AS total
+      FROM "efile_COAK_2016_E-Expenditure",
+        oakland_name_to_number
+      WHERE "Bal_Name" = "Measure_Name"
+      GROUP BY "Measure_Number", "Sup_Opp_Cd";
```

Comment:
You need to do the `Report_Num` shenanigans here so we don't get duplicate data

- [ ] add a DISTINCT ON clause for `("Measure_Number", "Sup_Opp_Cd")`
- [ ] add `Report_Num` to the GROUP BY
- [ ] add an ORDER BY `"Measure_Number", "Sup_Opp_Cd", "Report_Num" DESC`

...if I recall correctly